src/main/java/io/github/swagger2markup/internal/document/builder/MarkupDocumentBuilder.java

```diff
@@ -225,18 +225,19 @@ protected Type createInlineObjectType(Type type, String name, String uniqueName,
                 
                 MarkupDocBuilder descriptionContent = copyMarkupDocBuilder();
                 String description = defaultString(swaggerMarkupDescription(property.getDescription()));
-                if (isNotBlank(description))
+                if (isNotBlank(descriptionContent.toString())) 
```

Comment:
This should be already fixed with

https://github.com/jfiala/swagger2markup/commit/54e17321b6d65edb05af73526797063c92442013

see 
https://github.com/jfiala/swagger2markup/blob/master/src/main/java/io/github/swagger2markup/internal/document/builder/MarkupDocumentBuilder.java!

Best regards,

Johannes


Am 13.06.2016 um 16:56 schrieb Hugo de Paix de Coeur:
>
> In 
> src/main/java/io/github/swagger2markup/internal/document/builder/MarkupDocumentBuilder.java 
> <https://github.com/Swagger2Markup/swagger2markup/pull/156#discussion_r66806405>:
>
> > @@ -225,18 +225,19 @@ protected Type createInlineObjectType(Type type, String name, String uniqueName,
> >
> >                  MarkupDocBuilder descriptionContent = copyMarkupDocBuilder();
> >                  String description = defaultString(swaggerMarkupDescription(property.getDescription()));
> > -                if (isNotBlank(description))
> > +                if (isNotBlank(descriptionContent.toString()))
>
> This first test should test 'description' instead of 
> 'descriptionContent.toString()'.
> Other tests below are OK.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/Swagger2Markup/swagger2markup/pull/156/files/d37a374d023423c535cf325979f77bccdd63e0d0..44e33ad6f35f08a4974ac039399566e9400fa3f2#r66806405>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AD4ZDqAhGDLccgnbqrF3rD5Avd_zYxwxks5qLW-mgaJpZM4Izzvk>.
>

-- 
Mit freundlichen Grüßen,
Johannes Fiala
==========================
FWD GmbH
Viktoriagasse 7/33
1150 Wien
--------------------------
Tel: +43 1 595 23 55
Fax: +43 1 595 23 55-15
Web: http://www.fwd.at
==========================